charts/codesealer/README.md

```diff
@@ -183,43 +186,13 @@ To see what Codesealer helm parameters are available issue the following command
 helm show values codesealer/codesealer
 ```
 
-Codesealer has the following default settings which affect the injection method, Redis, and WAF:
-
-  --set initContainers.enabled=true \
-  --set ingress.nodePort.enabled=false \
-  --set worker.redis.service.name=redis-master \
-  --set worker.config.bootloader.redisUser=default \
-  --set worker.config.bootloader.redisUseTLS=false \
-  --set worker.config.bootloader.redisIgnoreTLS=true \
-  --set worker.config.endpoint.wafMonitorMode=false \
-  --set worker.config.endpoint.enableWaf=true \
-  --set worker.config.endpoint.wafFullTransaction=true \
-  --set worker.config.endpoint.crs.paranoiaLevel=1 \
-
-> NOTE: If you would like to install Codesealer in `enterprise` mode (with a local Manager) issue the
->       following commands:
->
-> ```bash
-> helm install codesealer ${CODESEALER_HELM_CHART} \
->   --create-namespace --namespace codesealer-system \
->   --set codesealerToken="${CODESEALER_TOKEN}" \
->   --set worker.ingress.namespace="${INGRESS_NAMESPACE}" \
->   --set worker.ingress.deployment="${INGRESS_DEPLOYMENT}" \
->   --set worker.ingress.port="${INGRESS_PORT}" \
->   --set worker.redis.namespace="${REDIS_NAMESPACE}" \
->   --set worker.config.bootloader.redisPassword="${REDIS_PASSWORD}" \
->   --set worker.replicaCount="${CODESEALER_WORKERS}" \
->   --set manager.enabled=true \
->   --wait --timeout=60s
-> ```
->
->  NOTE: To access local manager issue the following command:
->
-> ```bash
->kubectl port-forward service/core-manager -n ${INGRESS_NAMESPACE} 84444:8444 &
-> ```
-> You can access the manager at https://localhost:8444
->
+To access local manager issue the following command:
+
+ ```bash
+kubectl port-forward service/core-manager -n ${INGRESS_NAMESPACE} 84444:8444 &
```

Comment:
Two things:
* The manager is deployed in the `codesealer-system` namespace,
* We should mention how to get the admin password.
Basically, just copy what's in the 'NOTES.txt` about this.